Automattic/wp-calypso
# Help Center: Add tailored article list

Fixes https://github.com/Automattic/wp-calypso/issues/88707

## Proposed Changes

* This https://github.com/Automattic/wp-calypso/pull/88808 added a feature that allows tailored articles suggestions. This current PR is adding the article's structure to be suggested.
c/c @donalirl 

## Testing Instructions



* Using live link 
* Navigate to the pages (https://github.com/Automattic/wp-calypso/pull/89471/files#diff-1387095f7173d264c294187f77c20df1f285bad3ad7262e615718598da23ed65R1) 
* Open help center
* Make sure the articles suggested are matching the https://github.com/Automattic/wp-calypso/pull/89471/files#diff-1387095f7173d264c294187f77c20df1f285bad3ad7262e615718598da23ed65R1


## Pre-merge Checklist
